lorien/InfiniteCanvas/PanZoomCamera.gd

```diff
@@ -1,7 +1,9 @@
 extends Camera2D
 
 # -------------------------------------------------------------------------------------------------
+signal is_zooming(value: bool)
```

Comment:
I'm not sure about the `is_` naming convention for signals as it might be confused with a method. IMO signals should be past tense.

Reply:
Good point! I've renamed the signals now to `panning_toggled` and `zooming_toggled` respectively.